Plots/ColormapComparisons/CB_Rain.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun  4 14:12:34 2020
+
+@author: misi1684
+"""
+
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+from geocat.viz import util as gvutil
+import cartopy.feature as cfeature
```

Comment:
I would group the imports from cartopy together. It's a small thing, but it a bit more organized that way so people reading the example know what all they need to import from cartopy.